force_wfmanager/left_side_pane/workflow_tree.py

```diff
@@ -69,119 +71,25 @@
                                  enabled_when=call_modelview_editable)
 
 
+#: Wrapper to call workflow verification after each method
+def verify_wkflow(func):
```

Comment:
workflow

Reply:
I ended up changing from @verify_workflow to this because there's a function I use which is  also called verify_workflow. Is there any particular convention to go by for this sort of thing?

Reply:
ah damn... true.

triggers_verify then. 

The reason is that if you stay consistent with naming it's easier to grep around. Once you pick a name (e.g. workflow) stay consistent with it as much as possible, unless it's very encapsulated (e.g. a local variable called wf to store a workflow is fine)